dell-csi-helm-installer/csi-install.sh

```diff
@@ -13,13 +13,35 @@
 # limitations under the License.
 
 SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
-DRIVERDIR="${SCRIPTDIR}/../helm"
+DRIVERDIR="${SCRIPTDIR}/../"
+HELMCHARTAG="csi-vxflexos-2.8.0"
+
+if git ls-remote --quiet --tag  https://github.com/dell/helm-charts | grep -q "tags/${HELMCHARTAG}"; then
+  REPOREF="${HELMCHARTAG}"
+else
+  REPOREF="release-v1.8.0"
```

Comment:
release-v1.8.0 should not be hardcoded it should be passed from a cmd line parameter. based on the parameter it should be set